woodpecker-go/woodpecker/pipeline.go

```diff
@@ -11,3 +19,20 @@ func (c *client) PipelineQueue() ([]*Feed, error) {
 	err := c.get(uri, &out)
 	return out, err
 }
+
+// PipelineMetadata returns metadata for a pipeline, workflow name is optional.
+func (c *client) PipelineMetadata(repoID int64, pipelineNumber int, workflow ...string) ([]byte, error) {
```

Comment:
my only two questions I'd like to have opinions from another persion:
- the golang lib: should we copy the metadata types into the client and deserialize or just handle it via []byte as it's done now ...
- should we use the _optional_ feature like done now or do some other users missinterprete this as list ...